CBLClient/Apps/CBLTestServer-C/src/CollectionMethods.cpp

```diff
@@ -0,0 +1,427 @@
+#include "CollectionMethods.hpp"
```

Comment:
It looks great, a lot of new functions - thanks!

Will we be using them all? There's always the risk of functions being added and never used.

Reply:
We might not be using all of them.
These functions covers all the API in build number 150 for CBL-C